Notebook Examples for DP + GE: expect_column_values_vs_profile (#623)
capitalone/DataProfiler: examples/great_expecations/README.md

See a full list of Data Profiler Expectations available through Great Expectations on the [package page](https://greatexpectations.io/packages/capitalone_dataprofiler_expectations). 
If you are interested in learning more about expectations, here is a link to the [docs](https://greatexpectations.io/expectations).

### About these Notebooks
These notebook examples will utilize a dataset containing individual salary information across the world from the year 2020 until 2022.

> Each example will run through a different scenario that using the data and a custom expectation. 

Reviewer: ```suggestion
Each example will run through a different scenario that uses the data and a custom Data Profiler expectation. 
```

> These scenarios should give a practical understanding of how expectations can be used in real world contexts as data quality checks.

Reviewer: ```suggestion
These scenarios provide a practical example of how expectations can be used in real world contexts for data quality checks.
```
